pkgs/build-support/appimage/default.nix

```
@@ -4,11 +4,15 @@
 , lib, runCommand }:
 
 rec {
-  appimage-exec = pkgs.substituteAll {
+  appimage-exec = with pkgs; substituteAll {
```

```suggestion
  appimage-exec = substituteAll {
```

It's best to be explicit about the dependencies and add it to the function parameters instead.